src/app/assets/locales/locales.ts

```diff
@@ -2,7 +2,24 @@ import LocalizedStrings from "react-localization";
 
 export let strings = new LocalizedStrings({
   en: {
-    home: {},
+    home: {
+      hero: {
+        title: {
+          text1: "Unlock Your",
+          text2: "Digital Potential",
+        },
+        description: `Your one-stop digital software solutions company. At Tech, we thrive
+         on transforming your ideas into cutting-edge digital realities.`,
+         buttons:{
+          button1: "start a project",
+          button2: "out work"
+         },
+         altText: {
+          logo: "hero icon",
```

Comment:
Can you make the image alt more descriptive?

Reply:
done